helm-chart/values.yaml

```diff
@@ -210,6 +210,9 @@ cartDatabase:
     endpointPort: ""
     certificate: ""
 
+securityContext:
```

Comment:
Could we move this before the individual services values? So around line 58?

Reply:
Of course!